io/net/infiniband/mofed_install_test.py

```diff
@@ -99,9 +99,14 @@ def setUp(self):
         # on Avocado versions >= 50.0.  This is a temporary compatibility
         # enabler for older runners, but should be removed soon
         elif detected_distro.name in ['rhel', 'fedora', 'redhat']:
-            pkgs.extend(["make", "gcc", "python36-devel", "tcsh",
-                         "kernel-rpm-macros", "gdb-headless", "rpm-build",
-                         "gcc-gfortran", kernel_ver])
+            if detected_distro.version == "9":
+                pkgs.extend(["make", "gcc", "python3-devel", "tcsh",
```

Comment:
Instead of duplicating all package names you can restrict the if else only for python3/python36 package

            pkgs.extend(["make", "gcc", "tcsh",
                         "kernel-rpm-macros", "gdb-headless", "rpm-build",
                         "gcc-gfortran", kernel_ver])
            if detected_distro.version == "9":
                pkgs.extend(["python3-devel"])
            elif detected_distro.version == "8":
                pkgs.extend(["python36-devel"])